src/services/test-helpers/mock/data/getKeyNames.ts

```diff
@@ -0,0 +1,747 @@
+export const keyNames = {
```

Comment:
What would you need for this list to be automatically generated?

Reply:
I could generate off the Metadata, but would be a bit more code so I figured for the sake of keeping our sanity for these tests it would be easier to just copy and paste this.

Reply:
Fair enough.